includes/class-wc-payments-stripe-link-button-handler.php

```diff
@@ -83,6 +88,23 @@ public function display_button_html() {
 		<?php
 	}
 
+	/**
+	 * Indicates whether the Link express checkout button should be displayed.
+	 *
+	 * @return bool
+	 */
+	public function should_show_link_button() {
+		if ( ! WC_Payments_Features::is_link_enabled() ) {
+			return false;
+		}
+
+		if ( ! $this->is_checkout() || $this->is_pay_for_order_page() ) {
```

Comment:
Thanks for a great job while adding this additional statement, as it absolutely makes sense since we didn't implement a Link on the Pay for Order page. 

Question for myself to extend my knowledge: do you happen to know if we plan to support Link on the Pay for an Order page in the future? I guess the `$this->is_pay_for_order_page()` part comes from some acceptance criteria, but I'm unsure where to find these.

Reply:
I'm not aware of any plans to add support for Link or WooPay express buttons on the Pay for Order page. My assumption is that support just hasn't been added yet since these buttons are new additions.